rust/abacus-base/src/types/checkpoint_syncer.rs

```diff
@@ -54,10 +62,19 @@ pub struct MultisigCheckpointSyncerConf {
 
 impl MultisigCheckpointSyncerConf {
     /// Get a MultisigCheckpointSyncer from the config
-    pub fn try_into_multisig_checkpoint_syncer(&self) -> Result<MultisigCheckpointSyncer, Report> {
+    pub fn try_into_multisig_checkpoint_syncer(
+        &self,
+        origin: &str,
+        validator_checkpoint_index: IntGaugeVec,
+    ) -> Result<MultisigCheckpointSyncer, Report> {
         let mut checkpoint_syncers = HashMap::new();
         for (key, value) in self.checkpointsyncers.iter() {
-            checkpoint_syncers.insert(Address::from_str(key)?, value.try_into_checkpoint_syncer()?);
+            let gauge = validator_checkpoint_index
+                .with_label_values(&[origin, &key.to_lowercase()]);
```

Comment:
We have also been following the pattern to remove the `0x` prefix, not sure if this does that.

Reply:
Interesting, why is that the case? It seems to me that when referencing addresses, including the 0x prefix is pretty standard?

Reply:
No specific reason, so either we should change the rest or change this one.